### PR TITLE
Feature responsive desktop version

### PR DESCRIPTION
**Pull request by lewisjn-arch**
This PR request contains the following changes:

- [ ] I have grouped containers in html for desktop optimization.
- [ ] I have added media queries for the desktop version.
- [ ] I have made the page responsive upto a maximum width of 1440px.

NOTE: I am still configuring the container widths to respond to screens wider than 1440px.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,150 +50,167 @@ <h3 class="subject_text">I'm a Software Developer</h3>
             <li id="M_icon"><a href="#"><img src="Icons\Medium.png" alt="M-symbol link"></a></li>
         </ul>
         <div class="illustration"></div>
+        <div class="desktop_illustration"></div>
+        <div class="desktop_illustration2"></div>
+        <div class="desktop_illustration3"></div>
+        <div class="desktop_illustration4"></div>
+        <div class="desktop_illustration5"></div>
+        <div class="desktop_illustration_line"></div>
+        <div class="desktop_illustration_triangle"></div>
```
Comment (AlphaNtihinduka) [inline index.html]: - [ ] I have noticed that you are using different image backgrounds for an image that should be one. I suggest that you use only on background in the banner. This is to make sure that your portfolio is professional and easy to maintain.
- [ ] In addition, the background image should not have its `div` instead it should be used in the section or under HTML tag. For example for this particular case, the background image should be attached to `headline` ☺️.
Hint: You can consider exporting the illustration header which will download the whole image.
image

File: index.html [hunk 2]
```diff
@@ -40,150 +50,167 @@ <h3 class="subject_text">I'm a Software Developer</h3>
...
+                </button>
+            </a>
         </div>
-
-        <a href="#" class="resume_link">
-            <button type="button" class="resume_button">
-                <span class="link_text">Get my resume</span>
-            </button>
-        </a>
         <div class="about_me_illustration"></div>
```
Comment (AlphaNtihinduka) [inline index.html]: - As mentioned earlier about the background, the background image should not have its `div` instead it should be used in the section or under the HTML tag. For example for this particular case, the background image should be attached to the `about-me-container` or `about-me` ☺️.

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @lewisjn-arch 🙋‍♂️
Good job working on your desktop version. You have done the design to the best expectation and incredibly well. 👏  

You still need to implement a few on the project but you are almost there!!! 🚶 : 

TO HIGHLIGHT:
- Design matches the Figma design ✔️
- Desktop size starts at 768px ✔️
- Well-structured files ✔️

# STATUS: Required Changes :recycle: 

close


# Required Changes :recycle: 
- _Kindly consider the inline comments_ ☺️ 
- [ ] Please try as much as possible to ensure that your application does not have a horizontal scroll. The experience of scrolling horizontally can make the screen judder and create a headache-inducing feeling 😞 . Please fix this to have a professional portfolio ☺️ 
https://github.com/lewisjn-arch/Mobile-Portfolio-setup/assets/99394374/de78b7cc-c4d9-42e3-ae34-42b201d5962d



## Optional suggestions
N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: index.html (hunk 2 above)
Comment (lewisjn-arch) [reply index.html]: Thank you so much I thought I was so far from approval because it the desktop version isn't responsive beyond 1440px width

**Comment by lewisjn-arch** [comment]
This second review request has the following:

- [ ] I attached the background images to page containers using css.
- [ ] I changed some dimensions to their relative equivalents.


NOTE: I am suddenly getting a webhint error that I have failed to fix, I am getting an error that I do not understand in the details despite deleting and reinstalling linters files again.

**Review by shahadat3669 — CHANGES_REQUESTED** [review]
Hi @lewisjn-arch , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.



## Invalid Code Review Request

You have submitted a project, that is invalid because of linter errors.
- To ensure that your code adheres to the project's linting standards, it is important to have a green linter result. This means that the last commit in your pull request should have a green check mark (:white\_check\_mark:) or a summary indicating that all the required checks have passed. This helps in maintaining a clean and consistent codebase.

| Your | Professional |
| --- | --- |
| image | image |

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by lewisjn-arch** [comment]
This second re-review request has the following:

 - [ ] I attached the background images to page containers using css.
 - [ ] I changed some dimensions to their relative equivalents.


NOTE: I am suddenly getting a webhint error that I have failed to fix, I am getting an error that I do not understand in the details despite deleting and reinstalling linters files again.

Screenshot (18)

File: index.html [hunk 3]
```diff
@@ -39,152 +47,158 @@ <h3 class="subject_text">I'm a Software Developer</h3>
...
+                    no accounts or sign-ups required. Has been the 
+                    industry's standard.</p>
+                </div>
+                <div class="project_image"></div>
+                <ul class="project_skills">
+                    <li class="skill_container"><span class="skill">HTML</span></li>
+                    <li class="skill_container"><span class="skill">Bootstrap</span></li>
+                    <li class="skill_container"><span class="skill">Ruby</span></li>
+                </ul>
+                <button type="button" class="project_link"><span class="link_text">See project</span></button>
```
Comment (emmiiorji) [inline index.html:96]: [Optional] A close look at the figma design for the cards shows two displays for the cards. The first card with the "See Project" button shows how each card should look when hovered while the remaining cards without the "See Project" button shows how the cards will look without a hover.

Project card when hovered:
image

Project card without hover:
image

**Review by emmiiorji — APPROVED** [review]
Hi @lewisjn-arch, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Good Highlights✔️

- Requested changes have been implemented

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**